src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -164,7 +164,7 @@ export function transformVAOSAppointment(appt) {
     resourceType: 'Appointment',
     id: appt.id,
     status: appt.status,
-    cancellationReason: appt.cancellationReason,
+    cancelationReason: appt.cancelationReason,
```

Comment:
Always wondered about one 'L' or two 'LL' for cancelationReason. Is it consistently one 'L' now?

Reply:
Yeah, it should be just one l everywhere now. There are still instances of just "cancellation" in the code, but I think we can clean those up, too.